src/main/java/de/tum/cit/aet/artemis/quiz/repository/QuizSubmissionRepository.java

```diff
@@ -34,6 +34,14 @@ public interface QuizSubmissionRepository extends ArtemisJpaRepository<QuizSubmi
             """)
     Optional<QuizSubmission> findWithEagerResultAndFeedbackById(@Param("submissionId") long submissionId);
 
+    @Query("""
+                SELECT DISTINCT s
+                FROM QuizSubmission s
+                    LEFT JOIN FETCH s.submittedAnswers
+                WHERE s.participation.id IN :participationIds
+            """)
```

Comment:
This does not adhere to our commonly used formatting

```suggestion
    @Query("""
            SELECT DISTINCT s
            FROM QuizSubmission s
                LEFT JOIN FETCH s.submittedAnswers
            WHERE s.participation.id IN :participationIds
            """)
```